Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -39,9 +44,28 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def find_conflict_pr(cls, remote, branch):
-        for pr in remote.pull_requests.find(head=branch, opened=True):
-            return pr
+    def find_conflict_pr(cls, remote, radar_id):
+
+        radar = Tracker().client.radar_for_id(radar_id, additional_fields=['sourceChanges'])
+        shas = []
+
+        for entry in radar.sourceChanges.splitlines():
+            repo, action, sha = entry.split(', ')
+            if repo == 'WebKit':
+                shas.append(sha)
+
+        integration_branches = []
+        for prefix in ('ci', 'conflict'):
+            integration_branches.append(cls.INTEGRATION_BRANCH_PREFIX + "/" + prefix + "/" + shas[0][:10] + "_" + shas[-1][:10])
```

Comment:
.format string here?

Reply:
I think overall with opensource stuff we have tried to refrain from those python3 specifics things, unless that has changed recently @JonWBedard @briannafan

Reply:
.format is not a python3 thing, it is used lower in this file and in many others

Reply:
I think format strings are fine. f-strings are the python3 specific thing you're probably thinking of.